File: code-coaches/easy/izzy-the-iguana/izzy_the_iguana.py
def get_snack_points(snacks: list[str]) -> int:
    snacks_dict = {
        "Mango": 9,
        "Lettuce": 5,
        "Carrot": 4,
        "Cheeseburger": 0,
    }
    points = sum(snacks_dict.get(item, 0) for item in snacks)
    return points


def main():
    snacks = input().split()

    if get_snack_points(snacks) >= 10:
        print("Come on Down!")
    else:
        print("Time to wait")


if __name__ == "__main__":
    main()

File: code-coaches/easy/izzy-the-iguana/izzy_the_iguana.rb
def get_snack_points(snacks)
  snacks_dict = {
    "Mango" => 9,
    "Lettuce" => 5,
    "Carrot" => 4,
    "Cheeseburger" => 0,
  }
  points = snacks.sum { |item| snacks_dict[item] || 0 }
end

def main
  snacks = gets.chomp.split()

  if get_snack_points(snacks) >= 10
    puts "Come on Down!"
  else
    puts "Time to wait"
  end
end

if __FILE__ == $0
    main
end
